client/src/django.ts

```diff
@@ -17,10 +17,6 @@ interface Paginated<T> {
 const apiClient = axios.create({ baseURL: API_URL });
 let s3ffClient;
 
-apiClient.interceptors.response.use(null, (error) => {
-  const msg = error?.response?.data?.detail || 'No response from server';
-  throw new Error(msg);
-});
```

Comment:
Was this simply unnecessary?

Reply:
I moved it to the bottom of the file so that I could reference `djangoClient.logout()` without a lint error

Reply:
Oh I see, it's  still got this logic inside in addition to the new logic. Sorry for the noise.